config.default.json

```diff
@@ -14,8 +14,7 @@
     "entryNodes": [
       "MRCJTGX9x+PPiT3um1DQSHXvALaUg/tCH/oQr6mliGo=@35.246.104.243:14626"
     ],
-    "port": 14626,
-    "version": 1
```

Comment:
if we decide to add the networkID (by splitting the current `version`) we could set it here

Reply:
If the version number is connected to some other parameter like network-ID or Coordinator address, then I agree. But right now it would just be a parameter that you have to set to the right version or everything breaks. That was why I've removed it form the config for now.

Reply:
I meant something like instead of having 4 bytes for the protocol version we could have a field `networkID` (2 or 4 bytes) and reduce the `version` field to 1-2 bytes

Reply:
Do you think we should maybe remove the `version int` but introduce an `networdID string`?

Reply:
the version field could still be there just for defining the protocol version. Maybe 1 byte is enough for that? 256 possible different versions? For the `networkID` I like the idea of using `string`

Reply:
unfortunately, there is no `byte` type in protobuf, so we would have to use `uint32` or `int32`. But it uses variable length encoding, so it should not be much longer than a byte.

Reply:
it now has `uint32 version` and `uint32 networkID`